fibonanci.py

```diff
@@ -2,12 +2,12 @@
 # -*- coding: utf-8 -*-
 
 """
-This code is supposed to calculate the Fibonnaci sequence
+This code is supposed to calculate the FUCK Fibonnaci sequence
 and do some other calculations with it
 """
 
 # FIXME: Refactor the name for this library
-import matplotlib.pyplot as matplotlibpyplotlongnameSebiIhopeyoulikeit
+import matplotlib.pyplot as matplotlibpyplotlongnameSebiIhopeyoulikeit Sebi i'mgoing to kill u
```

Comment:
Remove the space, or ad a # before "Sebi i'm..."

Reply:
You can keep the comment in line 5 if you want

Reply:
Maybe I do it...